### ISSUE TITLE
Do we support different audio files and subtitles ?

### ISSUE THREAD

**Issue by bhupendra-cvent** [aws/amazon-ivs-react-native-player#145]
**Describe the bug**
We have a requirement to support a video with different audio and subtitles.
Do we have any way through which we can support this feature with Amazon-ivs-react-native-player?
If not, do we have any plan to develop this feature?

**Comment by maxstoller** [comment]
Hi @bhupendra-cvent: Is the video you are attempting to play generated through IVS? The IVS player only supports IVS content. We cannot guarantee compatibility/performance when playing 3P content.

**Comment by bhupendra-cvent** [comment]
Yes, video is generated through IVS, and multiple audios are embedded along with the video streaming URL. But subtitles are not embedded. They are fetched from different URLs.
Do we know how, if you guys have any plan to provide these features to support different audios and subtitles?
